deployment/template.env

```diff
@@ -8,6 +8,9 @@ JWT_SECRET=
 # If not set, it will use `http://localhost:8000`
 NEXT_PUBLIC_BACKEND_URL=
 
+# if using a nginx reverse proxy, set path here
+BACKEND_ROOT_PATH="/api"
+
 # Set password for user for `fullaccess` and `readonly` account to access content
 CONTENT_FULLACCESS_PASSWORD=
 CONTENT_READONLY_PASSWORD=
```

Comment:
These cannot be set to empty - set defaults here too? Or comment out so defaults get loaded from `app_config.py`.

Reply:
Same with JWT_SECRET?

Reply:
Seems like any var set to nothing here gets set to the empty string and causing issues later (`OPENAI_LLM_TYPE` too, for example). Should we add a note of this to the top of the file that all unused vars should be commented out?

Reply:
Good idea. Fixed